providers/implementations/kem/rsa_kem.c

```diff
@@ -265,6 +265,14 @@ static int rsasve_generate(PROV_RSA_CTX *prsactx,
             *secretlen = nlen;
         return 1;
     }
+
+#ifdef FIPS_MODULE
+    if (nlen < OPENSSL_RSA_FIPS_MIN_MODULUS_BITS/8) {
```

Comment:
rsakem_init() does a call to ossl_rsa_check_key(prsactx->libctx, vrsa, operation)

Reply:
Agreed